packages/gatsby-theme-docs/gatsby-ssr.js

```diff
@@ -22,4 +32,19 @@ export const onRenderBody = ({ setPostBodyComponents }) => {
         onLoad="function OptanonWrapper() {};"
       />,
     ]);
+
+  setHeadComponents([
```

Comment:
The logic here is based on the implementation of the gatsby-plugin-manifest: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-manifest/src/gatsby-ssr.js